app.py

```diff
@@ -45,23 +46,21 @@ def initialize(cls, _db):
     Conf.initialize(_db)
 
 
-def accepts_catalog(f):
+def accepts_auth(f):
     @wraps(f)
     def decorated(*args, **kwargs):
-        catalog = CatalogController(Conf.db).authenticated_catalog_from_request(
-            required=False
-        )
-        if isinstance(catalog, ProblemDetail):
-            return catalog.response
-        return f(catalog=catalog, *args, **kwargs)
+        server = authenticated_server_from_request(Conf.db, required=False)
```

Comment:
As I mentioned in my core review, referring to this as a 'server' is confusing. From our perspective, it's a client of an API. The fact that it also happens to be a server (a circ manager) doesn't matter in this context.